nixos/modules/services/web-apps/ocis.nix

```
@@ -0,0 +1,131 @@
+{
+  config,
+  lib,
+  pkgs,
+}:
```

Adding `...` here should fix it.

```suggestion
{ config, lib, pkgs, ... }:
```

---

Nope, still borked :(